package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "posthog-react-rrweb-player",
-  "version": "1.0.9",
+  "version": "1.0.10",
```

Comment:
So releasing a new version inside a branch is generally a bad idea. It causes issues if the package is ever used by anyone else and they update.

Rather versioning should be done purely off of master branch always.

To do local development, there are two options: `yalc` or using `'"dependency": "link:../posthog-react-rrweb-player"` when testing it out.